packages/astro/src/types/public/config.ts

```diff
@@ -961,6 +931,151 @@ export interface ViteUserConfig extends OriginalViteUserConfig {
 
 	server?: ServerConfig | ((options: { command: 'dev' | 'preview' }) => ServerConfig);
 
+	/**
+	 * @docs
```

Comment:
Will note that all these versions should be updated to 5.7.0, as the first *stable* release version they were included.